name.abuchen.portfolio/src/name/abuchen/portfolio/messages_de.properties

```diff
@@ -203,13 +204,13 @@ IBXML_Label = IB: Kontoumsatz-Flex-Query-Dateien
 
 InvestmentPlanAutoNoteLabel = Generiert von Sparplan ''{1}'' am {0}
 
-InvestmentPlanItemImportToolTip = Diese Buchung wird eine bereits durch einen Sparplan erstellte Buchung  ersetzen
+InvestmentPlanItemImportToolTip = Diese Buchung ersetzt eine bestehende Buchung, die durch einen Sparplan generiert wurde.
 
 IssueBuySellWithoutSecurity = ''{0}'' ohne Wertpapier
 
 IssueDividendWithoutSecurity = Dividende (Aussch\u00FCttung) ohne Wertpapier
 
-IssueInconsistentSharesHeld = Wertpapier ''{0}'' in Portfolio ''{1}'' hat negative Anteile: {2}
+IssueInconsistentSharesHeld = Wertpapier ''{0}'' im Portfolio ''{1}'' hat negative Anteile: {2}
```

Comment:
Again, fixing what wasn’t broken while not fixing what is (the term *Portfolio* was replaced by *Depot* long ago).

Reply:
Wertpapier "Addidas" **in** Portfolio "SuperDepot" hat negative Anteile: -75.63
Wertpapier "Addidas" **im** Portfolio "SuperDepot" hat negative Anteile: -75.63

Wenn das inkorrekt ist, dann entschuldige bitte.

Reply:
„Im“ ist nicht falsch, aber das ursprüngliche „in“ ist genausowenig falsch. Eindeutig falsch bzw. veraltet ist hingegen „Portfolio“.

Reply:
Da hast du natürlich vollkommen recht.
Ich würde die Überstetzung wiefolgt noch zum Kalenderfehler hinzufügen
Wertpapier ''{0}'' im Depot''{1}'' hat negative Anteile: {2}

Danke 👍🏻